### ISSUE TITLE
Any update for SwitchBot Keypad in the future?

### ISSUE THREAD

**Issue by bingo2233** [AdyRock/com.switchbot#51]

**Comment by AdyRock** [comment]
There is currently no API support for the keypad but I will add it to Homey as soon as it is there

**Comment by bingo2233** [comment]
Sounds good. ATM it's the only weatherproof keypad for usage outside that potentially could be used with Homey Pro.
Considering how many devices that are on the market its really strange when you think about it...

**Comment by bingo2233** [comment]
Hi AdyRock
Any update regarding API support for the keypad?

**Comment by AdyRock** [comment]
Nothing yet.

**Comment by robindelange** [comment]
Hi Ady, maybe I see it wrong, but I see keypad and keypad touch mentioned in the switchbot api. 

kind regards,

Robin de Lange

**Comment by AdyRock** [comment]
Yes, there are some features available such as creating a key, but nothing really useful like detecting when a code is used, etc.

**Comment by robindelange** [comment]
aah øke, thank you for the answer. let's hope the add the rest soon, even as the snapshot and lock night latch settings

**Comment by ReneVerwoerd** [comment]
Thanks, I’m hoping this will work soon as well! ☺️
Looks like an amazing addition to my Homey home.

**Comment by beltesassar** [comment]
I wanted to buy the Switchbot Keypad Touch, but I can't find the Keypad Touch in the list of supported devices. 
Is this Swithbot product still not supported by the app v.1.1.35 ?

**Comment by AdyRock** [comment]
Unfortunately, Switchbot still hasn't added any useful features to the API. 
It has been requested on their GitHub repo.
